kernel/Dockerfile

```diff
@@ -26,8 +25,7 @@ RUN apk add \
     tar \
     xz \
     xz-dev \
-    zlib-dev \
-    || true
```

Comment:
the `|| true` is there so  that you don't get whitespace changes when changing the last item or adding a new one ta the end.